### PR TITLE
BUG change the init for fracdev to let the sims converge

### PR DESCRIPTION
**Pull request by beckermr**
See my branch on my repo.

@NiallMac for viz

### CODE REVIEW

**Comment by esheldon** [comment]
How generic do you think this is?  Should we also adopt this for real data runs with deeper data for example? Is there something particular about the sims that is causing the problem?

**Comment by beckermr** [comment]
I don't know. Basically the code was starting at 1.5 which is the max for the old guessing array. The whole range of allowable fracdev values is tough. Allowing them outside of [0, 1] generally gives better fits IIRC but is unphysical at some level. 

I think we could try a small test patch in the data and see what happens.

**Comment by beckermr** [comment]
@esheldon I vote to merge this and deal with the consequences. We will never be able to properly test it all cases. :doitlive: :shipit:

**Comment by esheldon** [comment]
can we make it configurable, a keyword or part of a fracdev guesser or some such?

**Comment by esheldon** [comment]
also might be better to start from master since this is going to conflict

File: ngmix/bootstrap.py [hunk 1]
```diff
@@ -1840,7 +1840,8 @@ def __init__(self, obs,
                                         fracdev_grid['max'],
```
Comment (beckermr) [inline ngmix/bootstrap.py]: @esheldon It may already be configurable. IDK where `fracdev_grid` comes from.

Comment (beckermr) [reply ngmix/bootstrap.py]: If we can just change the config, then we should close this PR and move on.

**Comment by beckermr** [comment]
The merge conflicts are NBD.

File: ngmix/bootstrap.py [hunk 2]
```diff
@@ -1840,7 +1840,8 @@ def __init__(self, obs,
...
                                         fracdev_grid['num'])
         else:
-            self.fracdev_tests=linspace(-1.0,1.5,26)
+            # self.fracdev_tests=linspace(-1.0,1.5,26)
+            self.fracdev_tests=linspace(0.0,1.0,26)
```
Comment (beckermr) [inline ngmix/bootstrap.py]: @esheldon here!

Comment (esheldon) [reply ngmix/bootstrap.py]: ok I understand now

Comment (esheldon) [reply ngmix/bootstrap.py]: you can input the fracdev grid parameters, but this is the default.  So better to add something to the config file and then make ngmixer send it on to this bootstrapper

Comment (esheldon) [reply ngmix/bootstrap.py]: agreed we should update the default too though

**Comment by esheldon** [comment]
do you mind if I start a new PR with just this change to avoid the clutter from the whitespace changes?

File: ngmix/bootstrap.py (hunk 2 above)
Comment (beckermr) [reply ngmix/bootstrap.py]: ngmixer just sends everything straight through. It might already work. Let me check.

Comment (beckermr) [reply ngmix/bootstrap.py]: yeah ngmixer just sends it all straight in.

EG, https://github.com/esheldon/ngmixer/blob/master/ngmixer/bootfit.py#L786

**Comment by beckermr** [comment]
Shall we close this? I think the old code is old and we should move on.

**Comment by beckermr** [comment]
Closing. Feel free to reopen.

**Pull request closed without merging**